src/alert-manager/src/cluster-utilization/send_alert.py

```diff
@@ -44,131 +58,154 @@ def get_job_usage_info(job_usage_result, rest_url):
     job_list = resp.json()
 
     for v in job_usage_result["data"]["result"]:
-        job_infos[v["metric"]["job_name"]] = {"usage": v["value"][1][:6] + "%"}
+        job_infos[v["metric"]["job_name"]] = {
+            "job_name": v["metric"]["job_name"],
+            "usage": v["value"][1][:6] + "%"
+        }
+    for v in user_usage_result["data"]["result"]:
+        user_infos[v["metric"]["username"]] = {
+            "username": v["metric"]["username"],
+            "usage": v["value"][1][:6] + "%", "resources_occupied": 0
+        }
     for job_name, job_info in job_infos.items():
         url = urllib.parse.urljoin(rest_url + "/", job_name)
         resp = requests.get(url, headers=headers)
         if not resp.ok:
             logging.warning("request failed %s", resp.text)
+            del job_infos[job_name]
             continue
         resp_json = resp.json()
+        username = resp_json["jobStatus"]["username"]
         # get job duration
         if not resp_json["jobStatus"]["appLaunchedTime"]:
             logging.warning("job not start, ignore it")
+            del job_infos[job_name]
             continue
-        job_info["start_time"] = datetime.fromtimestamp(
+        job_infos[job_name]["start_time"] = datetime.fromtimestamp(
             int(resp_json["jobStatus"]["appLaunchedTime"]) / 1000,
             timezone.utc)
+        if job_infos[job_name]["start_time"] < datetime.now(timezone.utc) - timedelta(days = 7):
+            job_infos[job_name]["start_time"] = datetime.now(timezone.utc) - timedelta(days = 7)
```

Comment:
Why add this line? Please do not change the value of `start_time`. You can use another key for this

Reply:
> Why add this line? Please do not change the value of `start_time`. You can use another key for this

Added another key `start_time_in_7d`.